packages/components/src/DatePicker/DatePicker.spec.tsx

```diff
@@ -33,7 +33,7 @@ describe("<DatePicker />", () => {
 
   it("should have an empty input value when a date is not provided", () => {
     render(<DatePickerWrapper />)
-    const input = screen.getByLabelText("Input label")
+    const input = screen.getByLabelText("Input label", { selector: "input" })
```

Comment:
I've added the selector since the tests seemed to pick up on a partial match for the dialogue accessible title since it is concatenated to produce `Choose date for: {label}` - funnily enough even with `exact: true` passed into the options 🤷